test-e2e/test_basic_switch.py

```diff
@@ -16,12 +16,18 @@ def setUp(self):
 
     def tearDown(self):
         logger.info("Tearing down the test")
-        self.api.delete_pod("ep6")
-        self.api.delete_pod("ep7")
+        #self.api.delete_pod("ep6")
+        #self.api.delete_pod("ep7")
```

Comment:
We should delete the pods, and please choose different names other than ep6 and ep7 (e.g. test_name-ep-number)